server/notification.js

```diff
@@ -15,9 +17,21 @@ class Notification {
      * @returns {Promise<string>} Successful msg
      * Throw Error with fail msg
      */
-    static async send(notification, msg, monitorJSON = null, heartbeatJSON = null) {
+    static async send(notification, monitorName = null, monitorJSON = null, heartbeatJSON = null) {
```

Comment:
The function is not just for up/down.
I prefer to keep it as `msg`, because I will implement cert expiration notification.
And it possibly breaks the "Test" button.

Reply:
Hmm, I managed to find the "test" button implementation, and changed this to work with that, however I wasn't aware of cert expiry notifications.

I made this change as many of the other notification providers already customize this/don't use it, so it should assist in moving the content generation to functions/methods, but I can remove it from this PR now we've discussed that.